<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionNew_New_Database"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_New_Database"/>
   <addaction name="actionOpen_Database"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Item"/>
   <addaction name="actionRemove_Item"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_New_Database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new_database.png</normaloff>:/new_database.png</iconset>
   </property>
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAdd_Item">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/add_item.png</normaloff>:/add_item.png</iconset>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
  <action name="actionRemove_Item">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/remove_item.png</normaloff>:/remove_item.png</iconset>
   </property>
   <property name="text">
    <string>Remove Item</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
